### PR TITLE
[test] Integrate tests for f64x2 min/max/abs ops from WAVM

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

File: test/core/simd/meta/simd_f64x2.py [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+                [['0x00', '0x01'], ['0x00', '0x01']]
+            ],
+            [
+                [['0x02', '0x80000000'], ['0x02', '0x80000000']],
+                [['0x02', '0x80000000'], ['0x02', '0x80000000']]
+            ]
+        ]
+
+        # Assert data
+        lst_oprt_with_const_assert = {}
```
Comment (tlively) [inline test/core/simd/meta/simd_f64x2.py]: What does the `lst_oprt` here mean?

File: test/core/simd/meta/simd_f64x2.py [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+
+    NAN_NUMBERS = ('nan', '-nan', 'nan:0x4000000000000', '-nan:0x4000000000000')
+
+    def gen_test_fn_template(self):
+
+        # Get function code
+        template = Simdf32x4ArithmeticCase.gen_test_fn_template(self)
+
+        # Function template
+        tpl_func = '  (func (export "{}"){} (result v128) ({} {}{}))'
```
Comment (tlively) [inline test/core/simd/meta/simd_f64x2.py]: What does the `tpl` here mean? It also might be good to use names in the string interpolation so its clear what each part is supposed to be.

File: test/core/simd/meta/simd_f64x2.py [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+    @property
+    def combine_binary_arith_test_data(self):
+        return {
+            'min-abs': [
+                ['-1.125'] * 2, ['0.125'] * 2, ['0.125'] * 2
+            ],
+            'max-abs': [
+                ['-1.125'] * 2, ['0.125'] * 2, ['1.125'] * 2
+            ]
+        }
```
Comment (tlively) [inline test/core/simd/meta/simd_f64x2.py]: I'm not sure how these structures are used either. They seem different from the previous structures.

File: test/core/simd/meta/simd_f64x2.py [hunk 4]
```diff
@@ -0,0 +1,443 @@
...
+        for case_data in lst_oppo_signs_0:
+
+            if isinstance(case_data, str):
+                cases.append(case_data)
+                continue
+
+            cases.append(str(AssertReturn(case_data[0],
+                                          [self.v128_const(case_data[3][0], case_data[1][0]),
+                                           self.v128_const(case_data[3][1], case_data[1][1])],
+                                          self.v128_const(case_data[3][2], case_data[2][0]))))
```
Comment (tlively) [inline test/core/simd/meta/simd_f64x2.py]: This looks like it's missing a space of indentation.

File: test/core/simd/simd_f64x2.wast [hunk 5]
```diff
@@ -0,0 +1,2425 @@
...
+  (func (export "f64x2.max_with_const_18") (result v128) (f64x2.max (v128.const f64x2 0 1) (v128.const f64x2 0 2)))
+  (func (export "f64x2.max_with_const_19") (result v128) (f64x2.max (v128.const f64x2 2 -3) (v128.const f64x2 1 3)))
+  (func (export "f64x2.max_with_const_20") (result v128) (f64x2.max (v128.const f64x2 0 1) (v128.const f64x2 0 1)))
+  (func (export "f64x2.max_with_const_21") (result v128) (f64x2.max (v128.const f64x2 2 3) (v128.const f64x2 2 3)))
+  (func (export "f64x2.max_with_const_22") (result v128) (f64x2.max (v128.const f64x2 0x00 0x01) (v128.const f64x2 0x00 0x02)))
+  (func (export "f64x2.max_with_const_23") (result v128) (f64x2.max (v128.const f64x2 0x02 0x80000000) (v128.const f64x2 0x01 2147483648)))
+  (func (export "f64x2.max_with_const_24") (result v128) (f64x2.max (v128.const f64x2 0x00 0x01) (v128.const f64x2 0x00 0x01)))
+  (func (export "f64x2.max_with_const_25") (result v128) (f64x2.max (v128.const f64x2 0x02 0x80000000) (v128.const f64x2 0x02 0x80000000)))
+  ;; f64x2.max param vs const
+  (func (export "f64x2.max_with_const_27")(param v128) (result v128) (f64x2.max (local.get 0) (v128.const f64x2 0 1)))
```
Comment (tlively) [inline test/core/simd/simd_f64x2.wast]: Looks like there should be another space before the param here.

File: test/core/simd/meta/simd_f64x2.py (hunk 3 above)
Comment (Honry) [reply test/core/simd/meta/simd_f64x2.py]: These structures inherit from [simd_arithmetic.py](https://github.com/WebAssembly/simd/blob/7abe171dab76043a335282c4a447a873e471b64f/test/core/simd/meta/simd_arithmetic.py#L241), specific for combination tests. Here should have annotate to explain this.

**Pull request merged**